src/tasks/CloudFormationExecuteChangeSet/task.json

```diff
@@ -60,6 +60,22 @@
             "required": false,
             "helpMarkDown": "The stack name or Amazon Resource Name (ARN) of the stack associated with the change set. This value is required if you specified the name of a change set to execute. If the ARN of the change set was specified this field is optional.\n\nThe name must be unique in the region in which you are creating the stack. A stack name can contain only alphanumeric characters (case-sensitive) and hyphens. It must start with an alphabetic character and cannot be longer than 128 characters."
         },
+        {
+            "name": "noFailOnEmptyChangeSet",
+            "type": "boolean",
+            "label": "Do Not Fail On Empty Change Set",
+            "defaultValue": "false",
+            "required": false,
+            "helpMarkDown": "Causes the task to identify an empty change set and skip its excution without throwing an error."
+        },
+        {
+            "name": "deleteEmptyChangeSet",
+            "type": "boolean",
+            "label": "Delete Empty Change Set",
+            "defaultValue": "false",
+            "required": false,
+            "helpMarkDown": "Toggles the deletion of an empty change set.  Only used if `noFailOnEmptyChangeSet=true`."
```

Comment:
Adding a `"visibleRule": "noFailOnEmptyChangeSet = true"` entry should cover this case, in which case you should also remove the note here. Help note can probably read "Automatically delete change set if empty."

Reply:
Done